hazelcast-tpc-engine/src/main/java/com/hazelcast/internal/tpc/nio/NioEventloop.java

```diff
@@ -96,6 +59,40 @@ protected void run() throws Exception {
         } while (!stop);
     }
 
+    private int select() throws IOException {
```

Comment:
Revert.

Reply:
You mean throw exception instead of IOException or don't extract to a method?

Reply:
Inline that method :) I don't like to have this method pulled out. I can see all the logic of the eventloop on my screen without needing to dig into methods.